frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchAction.java

```diff
@@ -91,9 +93,9 @@ public void actionPerformed(AnActionEvent anActionEvent) {
 
       boolean isUpToDate = FetchUpToDateTimeoutStatus.isUpToDate(gitRepository);
       String fetchNotificationPrefix = isUpToDate
-          ? format(getString("action.GitMachete.BasePullBranchAction.notification.title.no-fetch-perform"),
-              FETCH_ALL_UP_TO_DATE_TIMEOUT_AS_STRING)
-          : format(getString("action.GitMachete.BasePullBranchAction.notification.title.fetch-perform"));
+          ? getString("action.GitMachete.BasePullBranchAction.notification.title.no-fetch-perform")
+              .format(FETCH_ALL_UP_TO_DATE_TIMEOUT_AS_STRING)
+          : getString("action.GitMachete.BasePullBranchAction.notification.title.fetch-perform").format();
```

Comment:
Huh `format` with no params makes no sense, unless I'm missing sth 🤔

Actually, pls open an issue to add an ArchUnit test to capture such nullary `format` calls